server/src/main/java/com/paypal/selion/grid/servlets/transfer/UploadRequestProcessor.java

```diff
@@ -295,40 +289,38 @@ private int parseRequestAsFileItems() throws FileUploadException {
         }
 
         private void populateHeadersMap() {
-            EnumMap<RequestHeaders, String> headersMap = getMapFromHeaders();
+            Map<String, String> headersMap = getRequestHeadersMap();
```

Comment:
I took a different approach to address this and assigned the result of calling `getManagedArtifactInstance().getRequestParameters()` to a member variable which is used throughout `UploadRequestProcessor`